packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -184,7 +184,7 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
 
         // Set the parent claim as countered. We do not need to append a new claim to the game;
         // instead, we can just set the existing parent as countered.
-        parent.countered = true;
+        parent.counteredBy = msg.sender;
```

Comment:
Should we allow people to set the address they want bonds paid to that are different from the source? Don't want to scope creep for a nice to have, but thinking of things like smart contract wallets or someone who's using a contract to batch multiple calls to the game.

Reply:
Can ticket this out; Imagine that most people will be able to wire up logic to extract the bond payment from the contract that submits them, but don't think it's important to add now.

Reply:
https://github.com/ethereum-optimism/client-pod/issues/434